lib/mongoid/timestamps/updated.rb

```diff
@@ -39,6 +39,16 @@ def set_updated_at
       def able_to_set_updated_at?
         !frozen? && !timeless? && (new_record? || changed?)
       end
+
+      # Set the updated at for a change that's already happened. This is used
```

Comment:
When I read the phrase "already happened" I think that this means the persistence operation has completed, which means the `updated_at` write below is not persisted with the association operation which seems wrong.

I'm not sure of the exact language to clarify this but another sentence regarding this would be appreciated.

Reply:
I mean, that is ultimately what is happening. The updated_at write is happening independently of the foreign key write (which is happening on the line right above it.)

I'm not sure how to tie them together...

Reply:
I'm referring to proxy.rb lines 32 and 33